csrc/src/types.cpp

```diff
@@ -0,0 +1,44 @@
+#include <torch/torch.h>
+#define LANTERN_TYPES_IMPL
+#include <lantern/types.h>
```

Comment:
local build fails with "Cannot open include file: 'lantern/types.h': No such file or directory". In `CMakeLists.txt` I changed
```
target_include_directories(styleganr PUBLIC
    ${PROJECT_SOURCE_DIR}/include
)
```
to
```
target_include_directories(styleganr PUBLIC
    ${PROJECT_SOURCE_DIR}/include
    ${TORCH_HOME}/include
)
```
so that it could find lantern, which seems to have resolved that issue.